convert/convert_test.go

```diff
@@ -0,0 +1,25 @@
+package convert
+
+import (
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/vcf"
+	"testing"
+)
+
+var seqA []dna.Base = dna.StringToBases("--TTTC--ATGAATAA")
```

Comment:
I had thought that the TODO comment in the code above mentioned that the reference sequence starting with gaps would not be handled well.

Reply:
Yes this is outdated, sequences starting with insertions now pass